api.go

```diff
@@ -1908,10 +1909,11 @@ type CreateSiteRuleBody struct {
 // ResponseSiteRuleBody contains the response from creating the rule
 type ResponseSiteRuleBody struct {
 	CreateSiteRuleBody
-	ID        string    `json:"id"`        //internal ID
-	CreatedBy string    `json:"createdby"` //Email address of the user that created the item
-	Created   time.Time `json:"created"`   //Created RFC3339 date time
-	Updated   time.Time `json:"updated"`   //Last updated RFC3339 date time
+	ID        string    `json:"id"`                //internal ID
+	CreatedBy string    `json:"createdby"`         //Email address of the user that created the item
+	Created   time.Time `json:"created"`           //Created RFC3339 date time
+	Updated   time.Time `json:"updated"`           //Last updated RFC3339 date time
+	Message   string    `json:"message,omitempty"` //Error message
```

Comment:
Why do we want this field in the body? Below it looks like we're returning the error message in the error value?

Reply:
At first i didn't realize we had a limit on the response code to be in the 400 range, so when I tested a value outside of that we actually get a 200 response code back. However it only had the message field populated, something like.
{"message":"Response code must be between 400 and 499"}.

Anyways the first condition was not enough to catch the error and I was returning an empty struct back.  The api really shouldn't switch data types on us without changing the response code, but that seems to be the case

Reply:
On closer inspection, I was wrong we were indeed getting a 400 back, but the client wasn't looking for PUT requests. I have removed message and added the check for put.